libs/core/src/components/pds-input/docs/pds-input.mdx

```diff
@@ -113,8 +113,82 @@ When `invalid` is true, the input displays the error message and provides a visu
+    webComponent: '<pds-input component-id="pds-input-prepend-append-example" label="Amount" type="text" value="100"><pds-select class="pds-input__prepend" slot="prepend"><option value="USD">USD</option><option value="EUR">EUR</option></pds-select><pds-select class="pds-input__append" slot="append"><option value="1">1</option><option value="2">2</option></pds-select></pds-input>'
+  }}>
+  <pds-input component-id="pds-input-prepend-append-example" label="Amount" type="text" value="100">
+    <pds-select class="pds-input__prepend" slot="prepend">
+      <option value="USD">USD</option>
+      <option value="EUR">EUR</option>
+    </pds-select>
+    <pds-select class="pds-input__append" slot="append">
+      <option value="1">1</option>
+      <option value="2">2</option>
+    </pds-select>
+  </pds-input>
+</DocCanvas>
+
+### Usage Guidelines
+
+- **Prefix/Suffix**: Use for small elements like icons or short text that should be visually integrated with the input field.
+- **Prepend/Append**: Use for larger elements like dropdowns or buttons that need to be visually separated from the input field.
+
+**Prefix Icon**
+<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+  <pds-icon name="mail" slot="prefix"></pds-icon>
+</pds-input>
+
+**Prepend Select**
+<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+  <pds-select class="pds-input__prepend" slot="prepend">
+    <option value="1">1</option>
+    <option value="2">2</option>
+    <option value="3">3</option>
+  </pds-select>
+</pds-input>
+
+**Prefix Icon and Append Select**
+<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+  <pds-icon name="dollar" slot="prefix"></pds-icon>
+  <pds-select class="pds-input__append" slot="append">
+    <option value="1">USD</option>
+    <option value="2">EUR</option>
+    <option value="3">GBP</option>
+  </pds-select>
+</pds-input>
+
+**Unstyled button suffix**
+<pds-input component-id="pds-input-prefix-suffix-example-2" label="Name" type="email" value="Frank Dux">
+  <pds-button class="pds-input__suffix" variant="unstyled" slot="suffix"><pds-icon name="remove-circle"></pds-icon></pds-button>
+</pds-input>
+
+**Box in prepend and append slots**
+<pds-input component-id="pds-input-prefix-suffix-example-2" label="Name" type="email" value="Frank Dux">
+  <pds-icon name="mail" slot="prefix"></pds-icon>
+  <pds-button class="pds-input__suffix" variant="unstyled" slot="suffix"><pds-icon name="remove-circle"></pds-icon></pds-button>
+</pds-input>
+
```

Comment:
Should we have an example that uses pds-text or text as well? I know sometimes we have inputs with like `https://` at the beginning or `mykajabi.com` at the end